lib/fs.js

```diff
@@ -1369,13 +1408,13 @@ function StatWatcher() {
   // the sake of backwards compatibility
   var oldStatus = -1;
 
-  this._handle.onchange = function(current, previous, newStatus) {
+  this._handle.onchange = function(newStatus) {
     if (oldStatus === -1 &&
         newStatus === -1 &&
-        current.nlink === previous.nlink) return;
+        statValues[2] === statValues[16]) return;
```

Comment:
It might be good to leave a comment to indicate that these values refer to `.nlink`